examples/react/start-basic-rsc/app/utils/posts.tsx

```diff
@@ -8,26 +8,28 @@ export type PostType = {
   body: string
 }
 
-export const fetchPost = createServerFn('GET', async (postId: string) => {
+export const fetchPost = async (postId: string) => {
+  'use server'
   console.log(`Fetching post with id ${postId}...`)
   await new Promise((r) => setTimeout(r, 500))
   const post = await axios
     .get<PostType>(`https://jsonplaceholder.typicode.com/posts/${postId}`)
     .then((r) => r.data)
     .catch((err) => {
       if (err.status === 404) {
-        throw notFound()
+        // throw notFound()
```

Comment:
Is `notFound` not working with streaming?

Reply:
This will be more complex to solve with RSCs, but we're punting on that for now.